components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -2082,6 +2084,8 @@ void RewardsServiceImpl::HandleFlags(const RewardsFlags& flags) {
   if (flags.persist_logs) {
     persist_log_level_ = kDiagnosticLogMaxVerboseLevel;
   }
```

Comment:
It would be nice if we could remove that, as the rest of the function operates on the `LedgerOptions` object. Do we still need `persist-logs`? If not, then we could also rename this function to, say, `ToLedgerOptions`:
```
auto options = ToLedgerOptions(RewardsFlags::ForCurrentProcess());
```

Reply:
I agree. For this PR, I'd like to keep the diff small. We should definitely improve this in a future PR.